lib/config/definitions.ts

```diff
@@ -140,6 +140,15 @@ const options: RenovateOptions[] = [
     admin: true,
     cli: false,
   },
+  {
+    name: 'onboardingCommitMessageBody',
```

Comment:
This seems like the wrong terminology. Look at how we configure `commitMessage` and `commitMessageBody` already. Anything new should be consistent with this.

Reply:
@rarkins I've updated the onboarding message to be made up of individual components like the normal `commitMessage`. I only included new options for `onboardingCommitMessagePrefix`, `onboardingCommitMessageAction`, and `onboardingCommitMessageTopic` as `extra` and `suffix` didn't seem necessary to me for an onboarding commit message. I also have it so `onboardingCommitMessagePrefix` will default to the value of `commitMessagePrefix` to make it more flexible. Let me know if this works for you.

Reply:
I was just meaning that you seem to be specifying `commitMessage` and not `commitMessageBody`. I don't think we have any reason to decompose the pieces further like we do with `commitMessage`.

Reply:
We construct like this:

```
commitMessage

commitMessageBody
```

Reply:
My bad @rarkins, when you said you look at how its configured I thought you wanted it decomposed. No harm though - I got to explore more of the code base :). If you're OK with what I had before, but simply renaming `onboardingCommitMessageBody` to `onboardingCommitMessage` I'm onboard. If you can confirm that is the right change I can make that update.

Reply:
Yes, I think that's the right semantics. And then existing prefix/semantic config should be fine to apply, as long as it's done consistently with how we apply it to `commitMessage`.

Reply:
Third times a charm :) 

I've pushed an update that includes the other two initial feedback points, reverts the onboarding message decomposition to what I had initially except that references to `onboardingCommitMessageBody` have been corrected to`onboardingCommitMessage`.